packages/@aws-cdk/cx-api/lib/cloud-assembly.ts

```diff
@@ -112,6 +113,25 @@ export class CloudAssembly {
     return artifact;
   }
 
+  /**
+   * Returns a metadata artifact from this assembly.
+   * @param id the identifier of the metadata artifact in this assembly
+   * @throws if there is no metadata artifact by that name
+   * @returns a `MetadataCloudArtifact` object.
+   */
+  public getMetadata(id: string): MetadataCloudArtifact {
```

Comment:
rename "metadata" with "tree" everywhere...

Reply:
Why do we need an `id` here? Isn't the ID always the same? I expect there to be only a single `tree.json` in the assembly.